install/jaspic/bin/ts.jte

```diff
@@ -104,7 +104,7 @@ orb.host=localhost
 orb.port=3699
 
 ########################################################################
-## Settings for Sun RI Java EE Implementation
+## Settings for Eclipse GlassFish Jakarta EE Implementation
```

Comment:
No comment as I'm not sure of what to change the RI reference to (I think that we eliminated the idea of a reference implementation from the Jakarta EE Platform but the TCKs still have the below ```RI``` properties).

Reply:
> Settings for a Jakarta EE Implementation that is treated like a reference implementation (RI)

For now that wouldn't work I think, as these settings are all very specific to GlassFish at this moment. It would certainly be doable eventually, but at the moment we don't seem to be able to use just any compatible implementation as "reference server".

Reply:
Very good point, I agree that only GlassFish would work currently.